Makefile

```diff
@@ -0,0 +1,25 @@
+all: deps clean install test
+
+.PHONY: docs
+
+install:
+	python setup.py build_ext
```

Comment:
My reason for using `build_ext` was to expose the verbose Cython output. Is there a way to get these warnings with `pip install`?